tests/staticfiles_tests/project/documents/cached/css/comments.css

```diff
@@ -0,0 +1,18 @@
+@import url("window.css");
+
+/* Single line: @import url("does.not.exist.A.css"); */
+
+/* Multi lines:
+ * @import url("does.not.exist.B.css");
+ */
+
+@font-face {
+  src: url("fonts/font.eot?#iefix") format("embedded-opentype");
+  /* Single line: src: url("does.not.exist.C.eot") format("embedded-opentype"); */
+}
```

Comment:
Can we add a test for inline comment? e.g.
```css
@font-face {
    src: /*url("does.not.exist.E.eot")*/ url("fonts/font.eot?#iefix") format("embedded-opentype");
}
```
and also for two comments in a line, e.g.
```css
@font-face {
    src: /*comment 1*/ url("fonts/font.eot?#iefix") /*comment 2*/ format("embedded-opentype");
}
```